### PR TITLE
New Device: Fortinet Fortigate 100EF

### PR DESCRIPTION
**Pull request by Nealtron**
Resubmitting PR https://github.com/netbox-community/devicetype-library/pull/1188.

Modified existing 100E config file to reflect the different interfaces between the 2 models.

### CODE REVIEW

File: FG-100EF.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+manufacturer: Fortinet
+model: FortiGate 100EF
+part_number: FG-100EF
+slug: fg-100ef
```
Comment (danner26) [inline FG-100EF.yaml:5]: please append `fortinet-` to the slug

File: FG-100EF.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+manufacturer: Fortinet
+model: FortiGate 100EF
+part_number: FG-100EF
+slug: fg-100ef
+is_full_depth: false
+u_height: 1
```
Comment (danner26) [inline FG-100EF.yaml:7]: Please add `weight` if you have/can find it

Comment (danner26) [inline FG-100EF.yaml:7]: Please add `airflow` if you have/can find it

Comment (danner26) [inline FG-100EF.yaml:7]: Please add a comment in markdown format linking to the manufacturer provided datasheet if you can find it

**Review by danner26 — CHANGES_REQUESTED** [review]

File: FG-100EF.yaml (hunk 1 above)
Comment (Nealtron) [reply FG-100EF.yaml:5]: Slug updated.

File: FG-100EF.yaml (hunk 2 above)
Comment (Nealtron) [reply FG-100EF.yaml:7]: Weight added.

Comment (Nealtron) [reply FG-100EF.yaml:7]: Airflow added.

Comment (Nealtron) [reply FG-100EF.yaml]: Not sure I did this part correctly. Could you verify?

Comment (danner26) [reply FG-100EF.yaml]: All good, I added it for you :)

**Pull request merged**